modules/microshift-auto-recovery-example-ostree-systems.adoc

```diff
@@ -4,15 +4,15 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="microshift-automation-example-ostree-systems_{context}"]
-= Automating the integration process with systemd for OSTree systems
+= Using auto recovery with {op-system-ostree}
+
+As a use case, consider the following example situation in which you want to automate the `auto-recovery` process for {op-system-ostree-first} systems that use systemd in the blueprint file.
 
 [IMPORTANT]
 ====
-You must include the entire `auto-recovery` process for OSTree systems that use `systemd` in the blueprint file.
+You must include the entire `auto-recovery` process for `rpm-ostree` systems that use `systemd` in the blueprint file.
```

Comment:
Is it really rpm-ostree system in the doc? I usually saw it being referred to as OSTree systems or RHEL For Edge (which apparently is different from RHEL Image Mode)

Reply:
Actually, I would call this RHEL for Edge. Image mode for RHEL is different, but one of the devs there said it is still using rpm-ostree, so....

Reply:
Yes, it uses (rpm-)ostree underneath